packages/builder/builder-rspack-provider/src/rspackPlugin/removeCssSourcemapPlugin.ts

```diff
@@ -0,0 +1,32 @@
+import type { Compiler, Compilation } from '@rspack/core';
+import type HtmlPlugin from '@rspack/plugin-html';
+import { COMPILATION_PROCESS_STAGE } from '@modern-js/builder-shared';
+
+export class RemoveCssSourcemapPlugin {
+  name: string;
+
+  htmlPlugin: typeof HtmlPlugin;
+
+  constructor(htmlPlugin: typeof HtmlPlugin) {
+    this.name = 'RemoveCssSourcemapPlugin';
+    this.htmlPlugin = htmlPlugin;
+  }
+
+  apply(compiler: Compiler) {
+    compiler.hooks.compilation.tap(this.name, (compilation: Compilation) => {
+      compilation.hooks.processAssets.tap(
+        {
+          name: this.name,
+          stage: COMPILATION_PROCESS_STAGE.PROCESS_ASSETS_STAGE_SUMMARIZE,
+        },
+        () => {
+          compilation.getAssets().forEach(asset => {
```

Comment:
`compilation.getAssets` will generate a huge overhead on Rspack. Would you please change it to `compilation.assets` related implementations?  https://github.com/web-infra-dev/rspack/blob/5990d5e5147b065517844b9a42b0b66297b36c57/packages/rspack/src/compilation.ts#L171

Reply:
how about `Object.keys(compilation.assets)`? Actually, I don't know the name of the css sourcemap output, so I need to enumerate assets and filter

Reply:
> how about `Object.keys(compilation.assets)`? Actually, I don't know the name of the css sourcemap output, so I need to enumerate assets and filter

Yes, using this will be fine!

Reply:
done~